deploy/bz1997062-crio-pid-leak/00-osd-cleanup-pids.DaemonSet.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: execpid-cleaner
+  namespace: kube-system
```

Comment:
Is there a reason we're running it in `kube-system`? That doesn't seem great

Reply:
that's taken directly from https://access.redhat.com/solutions/6304881